src/SpectatorView.Unity/Assets/SpatialAlignment.ASA/Scripts/SpatialAlignment/SpatialAnchorsCoordinateService.cs

```diff
@@ -334,6 +334,8 @@ private async Task<ISpatialCoordinate> TryCreateCoordinateImplAsync(Vector3 worl
             {
                 ReleaseSessionStartRequest();
             }
+
+            return null;
```

Comment:
>return null; [](start = 12, length = 12)

This seems like unreachable code, how would this ever be hit?

Reply:
product of removing a catch stub, fixing now.